sig-api-machinery/README.md

```diff
@@ -44,6 +44,7 @@ The following subprojects are owned by sig-api-machinery:
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/controller/garbagecollector/OWNERS
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/controller/namespace/OWNERS
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/quota/OWNERS
+    - https://raw.githubusercontent.com/kubernetes-sigs/kube-migrator/master/OWNERS
```

Comment:
How about kube-version-migrator? I'm worried that just kube-migrator is too vague.

Reply:
> How about kube-version-migrator? I'm worried that just kube-migrator is too vague.

`kube-storage-version-migrator`?  I wouldn't want someone to be confused into thinking that this will take care of general version upgrades.